plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestPostgreSqlConnectorTest.java

```diff
@@ -130,13 +140,131 @@ protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
                 return false;
 
             case SUPPORTS_CANCELLATION:
+            case SUPPORTS_UPDATE:
                 return true;
 
             default:
                 return super.hasBehavior(connectorBehavior);
         }
     }
 
+    @Override
+    public void testUpdate()
```

Comment:
Should we move these tests to `BaseJdbcConnectorTest`?

Reply:
So maybe we should move it to `BaseConnectorTest`, because now other connectors may also want to support UPDATE with metadata.

Reply:
Moved to BaseJdbcConnectorTest